platform/macos/joypad_macos.mm

```diff
@@ -228,7 +228,7 @@ @interface JoypadMacOSObserver ()
 @property(assign, nonatomic) BOOL isObserving;
 @property(assign, nonatomic) BOOL isProcessing;
 @property(strong, nonatomic) NSMutableDictionary<NSNumber *, Joypad *> *connectedJoypads;
-@property(strong, nonatomic) NSMutableArray<Joypad *> *joypadsQueue;
+@property(strong, nonatomic) NSMutableArray<GCController *> *joypadsQueue;
```

Comment:
For the record, the `joypad_ios.mm` implementation seems similar but doesn't specify the types for the `NSMutableArray` and `NSMutableDictionary` used here. Should that be harmonized?

Reply:
In case of iOS it should be:
```obj-c
@property(strong, nonatomic) NSMutableDictionary<NSNumber *, GCController *> *connectedJoypads;
@property(strong, nonatomic) NSMutableArray<GCController *> *joypadsQueue;
```

Probably worth merging both int a single file, it's using the same API and code should be almost same.

Reply:
Makes sense. I'd guess that should be for a follow-up 4.4 PR, as I'm evaluating whether this one should be merged for 4.3.